samples/client/petstore/java/jersey2-java6/docs/FakeApi.md

```diff
@@ -370,8 +382,10 @@ public class Example {
 
         FakeApi apiInstance = new FakeApi(defaultClient);
         FileSchemaTestClass body = new FileSchemaTestClass(); // FileSchemaTestClass | 
-        try {
+        try { 
             apiInstance.testBodyWithFileSchema(body);
+            
+            
```

Comment:
@zippolyte I'll see if I can get rid of these empty lines and the trailing space after `try {` in another PR.

Reply:
Filed https://github.com/OpenAPITools/openapi-generator/pull/5031

Reply:
Thanks for this ! I always find it tricky to format properly with the templating.